library/src/skins/default/src/sass/base/_icons.scss

```diff
@@ -188,6 +202,12 @@ $fa-font-path: "./fonts";
     warning: bi-exclamation-lg,
     expanded: bi-chevron-down,
     collapsed: bi-chevron-right,
+    sitestats-widget-visits : bi-person,
+    sitestats-widget-lessons : bi-book,
+    sitestats-widget-activity : bi-bar-chart,
+    sitestats-widget-resources : bi-folder,
+    sitestats-report : bi-file-text,
```

Comment:
Could we define it as a general purpose mapping like `report : bi-file-text`, also for `activity` and visits.